browser-test/src/support/admin_programs.ts

```diff
@@ -1495,7 +1495,7 @@ export class AdminPrograms {
   }
 
   getCommonIntakeFormToggle() {
-    return this.page.locator('input[name=isCommonIntakeForm]')
+    return this.page.locator('input[id=common-intake-checkbox]')
```

Comment:
(blocking) Please look this up by it's accessible name using `this.page.getByRole(...)`